go.mod

```diff
@@ -4,6 +4,9 @@ go 1.13
 
 require (
 	github.com/aws/aws-sdk-go v1.31.13
+	github.com/golang/mock v1.4.3
+	github.com/onsi/ginkgo v1.11.0
```

Comment:
You shouldn't need these two deps, `ginkgo` and `gomega` because you don't seem to be using BDD or the gomega matching.

Reply:
removed the `ginko` , `gomega` dependency